.github/make-release

```diff
@@ -49,6 +49,7 @@ test "$(jq .consistent < version.json)" = "true"
 
 # commit changes
 git add pom.xml modules/*/pom.xml assemblies/pom.xml assemblies/*/pom.xml
+git rm -f etc/workflows/fast.xml
```

Comment:
Actually :see_no_evil:…

This is part of the [Cut Release Tag](https://github.com/opencast/opencast/blob/develop/.github/workflows/cut-release-tag.yml) workflow, which is supposed to replace release managers manually cutting releases, but I never finished that. It still has a bug and I didn't fix that. But let me put that on the OSDN board.